hack/e2e.sh

```diff
@@ -73,9 +73,6 @@ backoff oc -n openshift-monitoring \
         --from-literal='config.yaml={"enableUserWorkload": true}' -o yaml --dry-run=client > /tmp/cluster-monitoring-config.yaml
 backoff oc apply -f /tmp/cluster-monitoring-config.yaml
 
-# Wait for user workload monitoring is deployed
-backoff oc -n openshift-user-workload-monitoring wait --for=condition=Ready pod -l app=thanos-ruler
```

Comment:
is there an example of what this flake looks like?  Because [the official docs][1] include a step:

> Check that the prometheus-operator, prometheus-user-workload and thanos-ruler-user-workload pods are running in the openshift-user-workload-monitoring project. It might take a short while for the pods to start...

and we probably want to follow that flow here.  Or tie into a workflow that enables user-workload monitoring for us (and performs the recommended wait?)?

[1]: https://docs.openshift.com/container-platform/4.8/monitoring/enabling-monitoring-for-user-defined-projects.html

Reply:
>we probably want to follow that flow here

I don't think we should be duplicating UWM verification in cincinnati e2e. If its broken QE would have noticed it, as we would simply retest it instead of report it as a bug.